src/Engine/Events/EventMap.cpp

```diff
@@ -22,7 +22,7 @@ EventMap EventMap::load(const Blob &rawData) {
     EventMap result;
 
     const char *pos = reinterpret_cast<const char *>(rawData.data());
-    const char *end = pos + rawData.size();
+    const auto end = pos + rawData.size();
```

Comment:
Same thing about almost always auto

Reply:
In this case I used it to avoid the ugly syntax for a constant pointer but can do either way.